### ISSUE TITLE
[Feature Request] Add an option to use the system default start button instead of ViStart's Orb

### ISSUE THREAD

**Issue by brunobits** [lee-soft/ViStart#28]
# Description
I'd like to have an option to use the system default start button instead of ViStart's Orb, since when using it with the Windows 11 taskbar or a custom taskbar such as [RetroBar](https://github.com/dremin/RetroBar/), the user will be seeing the taskbar and the ViStart orb ending up with two start buttons at once with the default start button being able to open the default system start menu.

# Demonstration
Windows 11 start button with ViStart's Orb on top of it.
![image](https://user-images.githubusercontent.com/48385307/221162708-4611d57c-f585-41b1-b487-6bb435197737.png)

Windows 10 start button with ViStart's Orb on top of it.
![image](https://user-images.githubusercontent.com/48385307/221163169-3b8d5f17-9b45-4448-8f89-21d7ed61d504.png)

RetroBar start button with ViStart's Orb on top of it.
![image](https://user-images.githubusercontent.com/48385307/221163726-a9d4dadd-9028-4bde-afd2-3b1b6fe5a972.png)

**Comment by GeorgeK1ng** [comment]
I don’t think we need feature to use system Start Orb as you can safely use custom one.
